bitnami/grafana-loki/templates/gateway/secret.yaml

```
@@ -20,6 +20,6 @@ metadata:
 type: Opaque
 data:
   {{- $password := (include "common.secrets.passwords.manage" (dict "secret" (include "grafana-loki.gateway.fullname" .) "key" "password" "providedValues" (list "gateway.auth.password") "context" $)) }}
-  password: {{ $password | quote }}
-  htpasswd: {{ htpasswd .Values.gateway.auth.username (b64dec $password) | b64enc | quote }}
+  password: {{ $password }}
```

The variable adds little value with your changes:
```suggestion
  password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "grafana-loki.gateway.fullname" .) "key" "password" "providedValues" (list "gateway.auth.password") "context" .) }}
```